src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -355,6 +355,11 @@ public async Task<string> AdjustSeatsAsync(Guid organizationId, int seatAdjustme
                 throw new NotFoundException();
             }
 
+            if (organization.Seats == null)
+            {
+                throw new BadRequestException("Organization is not seat limits, no need to adjust seats");
```

Comment:
This seems like it is missing a word. "using seat limits"?

Reply:
haha thanks. should be `limited`